diff --git a/spec/spec b/spec/spec
index ae3dbc3..c0d5004 100644
--- a/spec/spec
+++ b/spec/spec
@@ -711,6 +711,7 @@ beginseg
     include "$(BUILD_DIR)/src/code/z_overlay.o"
     include "$(BUILD_DIR)/src/code/z_play.o"
     include "$(BUILD_DIR)/src/code/z_play_hireso.o"
+    include "$(BUILD_DIR)/src/code/mc/mc_core.o"
     include "$(BUILD_DIR)/src/code/PreRender.o"
     include "$(BUILD_DIR)/src/code/TwoHeadGfxArena.o"
     include "$(BUILD_DIR)/src/code/TwoHeadArena.o"
@@ -5148,6 +5149,20 @@ beginseg
     include "$(BUILD_DIR)/src/overlays/actors/ovl_En_Rsn/ovl_En_Rsn_reloc.o"
 endseg
 
+beginseg
+    name "ovl_Mc_Intro"
+    compress
+    include "$(BUILD_DIR)/src/overlays/actors/ovl_Mc_Intro/z_mc_intro.o"
+    include "$(BUILD_DIR)/src/overlays/actors/ovl_Mc_Intro/ovl_Mc_Intro_reloc.o"
+endseg
+
+beginseg
+    name "ovl_Mc_Skyline"
+    compress
+    include "$(BUILD_DIR)/src/overlays/actors/ovl_Mc_Skyline/z_mc_skyline.o"
+    include "$(BUILD_DIR)/src/overlays/actors/ovl_Mc_Skyline/ovl_Mc_Skyline_reloc.o"
+endseg
+
 beginseg
     name "gameplay_keep"
     compress
